lib/mongoid/contextual/mongo.rb

```diff
@@ -182,9 +182,11 @@ def exists?(id_or_conditions = :none)
       # @example Explain the criteria.
       #   Band.where(name: "Depeche Mode").explain
       #
+      # @param [ Hash ] options customizable options (See Mongo::Collection::View::Explainable)
+      #
       # @return [ Hash ] The explain result.
-      def explain
-        view.explain
+      def explain(options = {})
+        view.explain(**options)
```

Comment:
Should just use:

```ruby
delegate :explain, to: :view
```

Reply:
since this class has `extend Forwardable`, it would need to be `def_delgator :view, :explain`

that work for you?

Reply:
updated but happy to change approach

Reply:
I’d say leave in the code comment like you had it above the def_delegator. Would be good to add spec demonstrating the options can be passed thru as well.

Im not a maintainer of this repo FYI.

Reply:
good call - done

anything else I should do to help get this merged?

Reply:
Looks good to me lets see what the MongoDB team says.